Libplanet/Net/ITransport.cs

```diff
@@ -32,6 +32,13 @@ public interface ITransport : IDisposable
         [Pure]
         DateTimeOffset? LastMessageTimestamp { get; }
 
+        /// <summary>
+        /// Whether this <see cref="ITransport"/> instance is running.
+        /// </summary>
+        /// <returns>Boolean value indicates whether the instance is running.</returns>
```

Comment:
I'm not sure to comment with `returns` XML tag for property. Is there the best practice you referenced or other code in our project? 🤔

Reply:
`NetMQTransport.Running` property was used to have comment with `returns` tag. I'm not sure it was correct or not.

Reply:
I see [the changes you said](https://github.com/planetarium/libplanet/pull/1215/files#diff-f8e579103886e10d7cb07b19770a9e7e8ef72415f3dba67a545715eb1dda7504L202-L205). I also don't have confident opinion for it. I just felt unnatural to use `returns` tag for `property`, something between `field` and `method`, not `method` (or `function`),

How do you think @planetarium/libplanet for this topic?

Reply:
Although `<returns>` can be used with properties spec-wise, we haven't used it for properties, but IMHO either way it's okay.